game_template/lib/main.dart

```diff
@@ -123,7 +123,7 @@ class MyApp extends StatelessWidget {
             GoRoute(
                 path: 'play',
                 pageBuilder: (context, state) => buildMyTransition<void>(
-                      key: ValueKey('play'),
+                      key: const ValueKey('play'),
```

Comment:
Hmm, was this flagged? We have a line in analysis_options like this:

```yaml
  prefer_const_constructors: false # Annoying in early development
```

Or was this a different, similar lint that was triggered? I still think requiring `const`s in early development is more pain than gain.

Reply:
reverted